tests/phpunit/Config/ApiTest.php

```diff
@@ -15,26 +17,36 @@
 use Keboola\GenericExtractor\Exception\UserException;
 use Keboola\GenericExtractor\Subscriber\AbstractSignature;
 use Keboola\Juicer\Client\RestClient;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\NullLogger;
 
 class ApiTest extends TestCase
 {
-    private function getClientMock(Request $request)
+    /**
+     * @return RestClient|MockObject
+     */
+    private function getClientMock(Request $request): RestClient
     {
         $beforeEventMock = self::createMock(BeforeEvent::class);
         $beforeEventMock->method('getRequest')->willReturn($request);
-        /** @var BeforeEvent $beforeEventMock */
+        /**
+ * @var BeforeEvent $beforeEventMock
```

Comment:
odsazení

Reply:
vdaka, nevsimol som si, ze mi to nejak rozbilo